### DISCUSSION TITLE
Tab specific session vs Browser specific session

### DISCUSSION THREAD

**Discussion by ravisumit33** [ravisumit33/Mysterio#56] [General]
Hey Guys !!

Since, Mysterio is a platform for anonymous chat we have thought of using sessions.
 
Django store session in cookie in which we can set expire time. One option which fits good is cookie expire on browser close. I can't find any option for cookie expire at tab close. 

This means that we have to ensure same state across tabs in one browser session.

On the browser side, I can find two options to store state. LocalStorage and SessionStorage.
LocalStorage is persistent and isn't automatically deleted on browser close. SessionStorage is tab specific. It is deleted everytime a tab is closed. If we want to maintain state across tabs, we need a mix of both options.

In my opinion , If we do not maintain state across tabs it will give bad UX as user have to enter details in every tab. So, what do you guys think? Should we maintain state across tabs or not?

@nileshvaishnav @ayushanshul07

**Comment by nileshvaishnav** [comment]
I think maintaining the state accross tabs is better.
